src/containers/AssignmentTexterContact.jsx

```diff
@@ -482,43 +527,84 @@ class AssignmentTexterContact extends React.Component {
 
   renderActionToolbar() {
     const { data, campaign, navigationToolbarChildren } = this.props
-
     const { contact } = data
+    const { messageStatus } = contact
 
-    return (
-      <Toolbar
-        style={inlineStyles.actionToolbar}
-      >
-        <ToolbarGroup
-          firstChild
-        >
-          <SendButton
-            threeClickEnabled={campaign.organization.threeClickEnabled}
-            onFinalTouchTap={this.handleClickSendMessageButton}
-            disabled={this.state.disabled}
-          />
-          {this.renderNeedsResponseToggleButton(contact)}
-          <RaisedButton
-            label='Canned responses'
-            onTouchTap={this.handleOpenPopover}
-          />
-          <RaisedButton
+    const size = $(window).width()
+    let toolBar = null
+
+   if (messageStatus === 'needsResponse' &&  size < 450 || messageStatus === 'messaged' && size < 450 ){
```

Comment:
could we achieve this with css breakpoints or max/min-width?  if too much a pain, then this is fine.